src/main.cpp

```diff
@@ -1807,6 +1807,7 @@ enum FlushStateMode {
 bool static FlushStateToDisk(CValidationState &state, FlushStateMode mode) {
     LOCK(cs_main);
     static int64_t nLastWrite = 0;
+    try {
```

Comment:
@sipa Why did you choose to leave out the indentation? Our code get's pretty ugly again over time? Could we finally let the clang-format tool let do it's job and we also had a remove orphan spaces and tabs script if I remember correctly...